canvas_data_warehouse/views.py

```diff
@@ -8,14 +8,18 @@
 
 class CanvasDataViewSet(viewsets.ViewSet):
     '''create API views for getting graph data'''
-    @list_route(name='get-canvas-data')
+    @list_route(url_path='data')
     def get_canvas_data(self, request):
-        '''getting data-graph data'''
-        CanvasDataService.dispatch(request.GET.get('graph_type'), request.GET)
-        return Response(request.GET, status=status.HTTP_200_OK)
+        '''getting canvas-data'''
+        graph_type = request.GET.get('graph_type')
+        if graph_type is None:
+            return Response("错误的参数格式", status=status.HTTP_400_BAD_REQUEST)
```

Comment:
We already have a helper class `BadRequest` to raise the exception, see `infra/exceptions.py`. e.g.:

```python
raise exceptions.BadRequest(<description_text_here>)
```